### ISSUE TITLE
[BUG][Ender 3 v2][dwin] MYSERIAL1' was not declared in this scope

### ISSUE THREAD

**Issue by CRCinAU** [MarlinFirmware/Marlin#20761]
This seems to be introduced since yesterday, but I haven't found a commit yet that has broken this in bugfix-2.0.x

Possible its a Configuration update that's broken it. I note that the Configurations repo had a push ~14 hours ago - but as its a force / smooshed commit, there's no history to compare.

Configs: https://github.com/MarlinFirmware/Configurations/tree/bugfix-2.0.x/config/examples/Creality/Ender-3%20V2

```
Marlin/src/lcd/dwin/dwin_lcd.cpp: In function 'void DWIN_Send(size_t&)':
Marlin/src/lcd/dwin/dwin_lcd.cpp:85:20: error: 'MYSERIAL1' was not declared in this scope
   LOOP_L_N(n, i) { MYSERIAL1.write(DWIN_SendBuf[n]); delayMicroseconds(1); }
                    ^~~~~~~~~
Marlin/src/lcd/dwin/dwin_lcd.cpp:85:20: note: suggested alternative: 'MYSERIAL0'
   LOOP_L_N(n, i) { MYSERIAL1.write(DWIN_SendBuf[n]); delayMicroseconds(1); }
                    ^~~~~~~~~
                    MYSERIAL0
Marlin/src/lcd/dwin/dwin_lcd.cpp:86:20: error: 'MYSERIAL1' was not declared in this scope
   LOOP_L_N(n, 4) { MYSERIAL1.write(DWIN_BufTail[n]); delayMicroseconds(1); }
                    ^~~~~~~~~
Marlin/src/lcd/dwin/dwin_lcd.cpp:86:20: note: suggested alternative: 'MYSERIAL0'
   LOOP_L_N(n, 4) { MYSERIAL1.write(DWIN_BufTail[n]); delayMicroseconds(1); }
                    ^~~~~~~~~
                    MYSERIAL0
Marlin/src/lcd/dwin/dwin_lcd.cpp: In function 'bool DWIN_Handshake()':
Marlin/src/lcd/dwin/dwin_lcd.cpp:97:10: error: 'MYSERIAL1' was not declared in this scope
   while (MYSERIAL1.available() > 0 && recnum < (signed)sizeof(databuf)) {
          ^~~~~~~~~
Marlin/src/lcd/dwin/dwin_lcd.cpp:97:10: note: suggested alternative: 'MYSERIAL0'
   while (MYSERIAL1.available() > 0 && recnum < (signed)sizeof(databuf)) {
          ^~~~~~~~~
          MYSERIAL0
*** [.pio/build/STM32F103RET6_creality/src/src/lcd/dwin/dwin_lcd.cpp.o] Error 1
```

**Comment by thinkyhead** [comment]
This will be solved when #20738 is merged.

**Comment by CRCinAU** [comment]
Can confirm - is working ok now.
